lib/SPIRV/libSPIRV/SPIRV.debug.h

```diff
@@ -1021,7 +1021,10 @@ inline bool hasDbgInstParentScopeIdx(const uint32_t Kind,
     ParentScopeIdx = LocalVariable::ParentIdx;
     return true;
   case SPIRVDebug::ImportedEntity:
-    ParentScopeIdx = ImportedEntity::ParentIdx;
+    if (ExtKind == SPIRV::SPIRVEIS_NonSemantic_Shader_DebugInfo_200)
```

Comment:
I guess we should check for both NonSemantic specs, because this instruction is added in Shader.100 spec https://github.com/KhronosGroup/SPIRV-Registry/blob/main/nonsemantic/NonSemantic.Shader.DebugInfo.100.asciidoc#DebugImportedEntity
Not sure why the test passes though..

Reply:
Applied